assembl/static2/js/app/pages/idea.jsx

```diff
@@ -54,8 +54,35 @@ const noRowsRenderer = () => {
 };
 
 class Idea extends React.Component {
+  topPostsToLoad = (topPosts, edges) => {
+    const { hash } = window.location;
+    if (hash !== '') {
+      const id = hash.replace('#', '');
+      const post = edges.filter((edge) => {
```

Comment:
use `.find` instead of `.filter` if it is for only one element : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
find and findIndex are much better indeed.